docs/content/terraform.md

```diff
@@ -219,39 +507,40 @@ touches_iam {
 ####################
 
 # list of all resources of a given type
-instance_names[resource_type] = all {
+resources[resource_type] = all {
     some resource_type
     resource_types[resource_type]
     all := [name |
-        tfplan[name] = _
-        startswith(name, resource_type)
+        name:= tfplan.resource_changes[_]
+        name.type == resource_type
     ]
 }
 
-# number of deletions of resources of a given type
-num_deletes[resource_type] = num {
+# number of creations of resources of a given type
+num_creates[resource_type] = num {
     some resource_type
     resource_types[resource_type]
-    all := instance_names[resource_type]
-    deletions := [name | name := all[_]; tfplan[name]["destroy"] == true]
-    num := count(deletions)
+    all := resources[resource_type]
+    creates := [res |  res:= all[_]; res.change.actions[_] == "create"]
```

Comment:
I am not sure if it's the best way to compute the number of created resources because `creates`  array contains the content of the resource and we don't use it anymore. 

What do you think ?